framework/test/unit/modules/interoperability/mainchain/commands/sidechain_registration.spec.ts

```diff
@@ -160,22 +212,25 @@ describe('Sidechain registration command', () => {
 			const result = await sidechainRegistrationCommand.verify(verifyContext);
 
 			expect(result.status).toBe(VerifyStatus.FAIL);
-			expect(result.error?.message).toInclude(
-				'Name substore must not have an entry for the store key name',
-			);
+			expect(result.error?.message).toInclude('Name already registered.');
 		});
 
-		it('should return error if store key networkID already exists in networkID store', async () => {
-			await networkIDSubstore.set(createStoreGetter(stateStore), networkID, {
-				id: utils.intToBuffer(0, 4),
+		it('should return error if store key chainID already exists in chainID store', async () => {
```

Comment:
i think cases for 
* ` trs.params.chainID[0] != CHAIN_ID_MAINCHAIN[0]:`
* sidechainRegistrationFee !== REGISTRATION_FEE
* vailableBalance < REGISTRATION_FEE
are missing?